e2e/app.e2e-spec.ts

```diff
@@ -1,14 +1,23 @@
+import { browser } from 'protractor';
 import { KetohubPage } from './app.po';
 
 describe('ketohub App', () => {
   let page: KetohubPage;
 
   beforeEach(() => {
+    browser.waitForAngularEnabled(false);
```

Comment:
Can you add a comment that notes that this is a workaround due to a conflict with angularfire? It there's a specific Github issue we can link to to, that would be helpful to include as well.